demo/Titanic/simulation/ocean.py

```diff
@@ -5,7 +5,7 @@
 
 SECONDS_IN_MINUTE  = 60
 COLLISION_INTERVAL = 10  # Following should hold True; SECONDS_IN_MINUTE % COLLISION_INTERVAL == 0
-
```

Comment:
Not sure if this is the cause, but collision detection is way more accurate than during my last demo. Good improvement!

Reply:
Part of the fix indeed :)